scripts/library_tools.py

```
@@ -0,0 +1,761 @@
+            draw.text((x + 4, y + cell // 2), "missing", fill="red", font=font)
+            continue
+
+        try:
+            png_bytes = cairosvg.svg2png(
+                url=str(svg_path),
+                output_width=cell,
+                output_height=cell,
+            )
+            img = Image.open(BytesIO(png_bytes)).convert("RGB")
+            canvas.paste(img, (x, y))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)